### PR TITLE
Add messaging around `init --site`

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: packages/wrangler/src/index.tsx [hunk 1]
```diff
@@ -309,6 +320,18 @@ export async function main(argv: string[]): Promise<void> {
         }
         throw new CommandLineArgsError(message);
       }
+      if (args.site) {
+        const message =
+          "The --site option is no longer supported.\n" +
+          "If you wish to create a brand new Worker Sites project then clone the `worker-sites-template` starter repository:\n\n" +
+          "```\n" +
+          "git clone --depth=1 --branch=wrangler2 https://github.com/cloudflare/worker-sites-template my-site\n" +
+          "cd my-site\n" +
```
Comment (threepointone) [inline packages/wrangler/src/index.tsx:329]:
```suggestion
          "git clone --depth=1 --branch=wrangler2 https://github.com/cloudflare/worker-sites-template ${args.name || "my-site"}\n" +
          "cd ${ args.name || "my-site" }\n" +
```

File: packages/wrangler/src/index.tsx [hunk 2]
```diff
@@ -264,14 +264,19 @@ export async function main(argv: string[]): Promise<void> {
           default: "worker",
         })
         .positional("template", {
-          describe: "a link to a GitHub template",
+          describe: "The URL of a GitHub template",
           default: "https://github.com/cloudflare/worker-template",
```
Comment (threepointone) [inline packages/wrangler/src/index.tsx:268]: won't we need a w2 version of this repository as well?

**Review by threepointone — APPROVED** [review]
If the template has w1 placeholders then these commands won't be enough, but I guess it's all we can do now. Stamping this.

File: packages/wrangler/src/index.tsx (hunk 1 above)
Comment (petebacondarwin) [reply packages/wrangler/src/index.tsx:329]: Nice idea!

File: packages/wrangler/src/index.tsx (hunk 2 above)
Comment (petebacondarwin) [reply packages/wrangler/src/index.tsx:268]: Really, for the default, we should just encourage the use of `wrangler init`.

Comment (petebacondarwin) [reply packages/wrangler/src/index.tsx:268]: If we are hitting the default then we should just encourage the use of `wrangler init`, right?.

File: packages/wrangler/src/index.tsx (hunk 1 above)
Comment (petebacondarwin) [reply packages/wrangler/src/index.tsx:329]: Nice idea!

**Pull request merged**